hugo/data/sref_locations.yaml

```
@@ -61,5 +61,6 @@ post: [algorithms, proof_of_spacetime]
 election_post: [algorithms, proof_of_spacetime, election_post]
 
 # nodes
+clock: [filecoin_blockchain]
```

does it work? no need for
```suggestion
clock: [systems, filecoin_nodes, clock]
```
?